extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala

```diff
@@ -132,6 +134,34 @@ class IdentifierTableExtractor extends TableExtractor {
   }
 }
 
+class IcebergCallCommandExtractor extends TableExtractor {
+  override def apply(spark: SparkSession, callStatementArgs: AnyRef): Option[Table] = {
+    val literals = callStatementArgs.asInstanceOf[Seq[Literal]]
+    if (literals.isEmpty) {
+      throw new IllegalArgumentException(
+        s"Unknown parameters in iceberg call command: $callStatementArgs")
+    }
+    // iceberg will rearrange the parameters according to the parameter order
+    // defined in the procedure, where the table parameters are currently always the first.
+    lookupExtractor[IcebergTableExtractor].apply(spark, literals.head)
```

Comment:
iceberg will rearrange the parameters according to the parameter order defined in the procedure, where the table parameters are currently always the first.
For example `RewriteDataFilesProcedure#PARAMETERS` and in `RewriteDataFilesProcedure#call` take the first parameter.